db/models.py

```diff
@@ -31,22 +34,80 @@ class CinemaHall(models.Model):
     rows = models.IntegerField()
     seats_in_row = models.IntegerField()
 
-    @property
-    def capacity(self) -> int:
-        return self.rows * self.seats_in_row
-
     def __str__(self) -> str:
         return self.name
 
 
 class MovieSession(models.Model):
+    movie = models.ForeignKey(Movie, on_delete=models.CASCADE)
     show_time = models.DateTimeField()
-    cinema_hall = models.ForeignKey(
-        to=CinemaHall, on_delete=models.CASCADE, related_name="movie_sessions"
-    )
-    movie = models.ForeignKey(
-        to=Movie, on_delete=models.CASCADE, related_name="movie_sessions"
+    cinema_hall = models.ForeignKey(CinemaHall, on_delete=models.CASCADE)
+
+    def __str__(self) -> str:
+        show_time_str = self.show_time.strftime('%Y-%m-%d %H:%M:%S')
+        return f"{self.movie.title} {show_time_str}"
+
+
+class User(AbstractUser):
+    pass
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        get_user_model(),
+        on_delete=models.CASCADE,
+        related_name="orders"
     )
```

Comment:
Code Style: According to the checklist, you should implement `class Meta` only after you defined all attributes. In your case, the `class Meta` is defined before some attributes. It's a good practice to keep the `class Meta` at the end of the model declaration. This will improve the readability of your code.